azure_arc_k8s_jumpstart/cluster_api/gitops/basic/az_k8sconfig_capi.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+echo "Exporting environment variables"
+export appId='<Your Azure service principal name>'
+export password='<Your Azure service principal password>'
+export tenantId='<Your Azure tenant ID>'
+export resourceGroup='<Azure resource group name>'
+export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appClonedRepo='<The URL for the "Azure Arc Jumpstart" forked GitHub repository>'
+export namespace='hello-arc'
+
+
+# Installing Helm 3
+echo "Installing Helm 3"
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+
+# Installing Azure CLI & Azure Arc Extensions
+echo "Installing Azure CLI"
+curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+
+az extension add --name connectedk8s
+az extension add --name k8s-configuration
+
+# Login to Azure
+echo "Log in to Azure with Service Principal"
+az login --service-principal --username $appId --password $password --tenant $tenantId
+
```

Comment:
Let's add a code block to add the Arc K8s resource providers registration to avoid it as a prereq. See #921 as an example for servers.

Reply:
Added this, let me know if anything else.
# Registering Azure Arc providers
echo "Registering Azure Arc providers"
az provider register --namespace Microsoft.Kubernetes --wait
az provider register --namespace Microsoft.KubernetesConfiguration --wait
az provider register --namespace Microsoft.ExtendedLocation --wait

az provider show -n Microsoft.Kubernetes -o table
az provider show -n Microsoft.KubernetesConfiguration -o table
az provider show -n Microsoft.ExtendedLocation -o table